packages/broker/src/plugins/operator/OperatorFleetState.ts

```diff
@@ -37,6 +31,7 @@ export class OperatorFleetState extends EventEmitter<OperatorFleetStateEvents> {
     private readonly heartbeatIntervalInMs: number
     private readonly latencyExtraInMs: number
     private readonly heartbeatTimestamps = new Map<NodeId, number>()
```

Comment:
Could simplify the state handling by combining `heartbeatTimestamps` and `peerDescriptors` to a single private `readonly latestHeartbeats = new Map<NodeId, number>()`

```
interface Heartbeat {
    timestamp: number
    peerDescriptor: NetworkPeerDescriptor
}
```

Reply:
Fixed in d2887938881ae8400c0d35b0e33de7ecce795b39